dockerfile

```diff
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.8-slim-buster
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+COPY . .
+
+CMD [ "python3", "-m" , "flask", "run", "--host=0.0.0.0"]
```

Comment:
TIL that there is a set of naming conventions in place on `hub.docker.com` for Dockerfiles with relation to their autobuild service. We build our own images, so this isn't a big deal to us, but I think it may be worth `git mv`ing this file to change its name to Dockerfile. I generally see it capitalized FWIW. Not a big deal by any means though.